templates/project_root_containers/passive/MyGeoCave.cxx

```diff
@@ -20,108 +20,120 @@
 
 #include "MyGeoCave.h"
 
-#include "FairGeoBasicShape.h"          // for FairGeoBasicShape
-#include "FairGeoMedia.h"               // for FairGeoMedia
-#include "FairGeoMedium.h"              // for FairGeoMedium
-#include "FairGeoNode.h"                // for FairGeoNode, etc
-#include "FairGeoShapes.h"              // for FairGeoShapes
+#include "FairGeoBasicShape.h"   // for FairGeoBasicShape
+#include "FairGeoMedia.h"        // for FairGeoMedia
+#include "FairGeoMedium.h"       // for FairGeoMedium
+#include "FairGeoNode.h"         // for FairGeoNode, etc
+#include "FairGeoShapes.h"       // for FairGeoShapes
 
-#include <TList.h>                      // for TList
-
-#include <string.h>                     // for strcmp
-#include <iostream>                     // for cout
+#include <TList.h>    // for TList
+#include <iostream>   // for cout
+#include <string.h>   // for strcmp
 
 using namespace std;
-ClassImp(MyGeoCave)
+ClassImp(MyGeoCave);
 
 MyGeoCave::MyGeoCave()
-  : FairGeoSet(),
-    name("cave")
+    : FairGeoSet()
+    , name("cave")
 {
-  // Constructor
-  fName="cave";
-  name="cave";
-  maxModules=1;
+    // Constructor
+    fName = "cave";
+    name = "cave";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'name' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/FairRootGroup/FairRoot/pullRequest?prid=5323596)